catalog/app/containers/Admin/Buckets/Buckets.tsx

```diff
@@ -766,6 +767,12 @@ interface PrimaryCardProps {
 function PrimaryCard({ bucket, className, disabled, onSubmit }: PrimaryCardProps) {
   const initialValues = bucketToPrimaryValues(bucket)
   const ref = React.useRef<HTMLElement>(null)
+  const { urls } = NamedRoutes.use()
+  const configPath = '.quilt/catalog/config.yaml'
```

Comment:
this constant looks strange here in the function body, especially considering we already have it in `constants/quiltConfigs`